winui/Masked-TextBox/Customization.md

```diff
@@ -7,4 +7,81 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Customization in WinUI SfMaskedTextBox
\ No newline at end of file
+# Customization in WinUI MaskedTextBox
+
+This section explains about header, header template and description properties of MaskedTextBox.
+
```

Comment:
@Vignesh-SF3580 This section explains the header, header template, and description of MaskedTextBox.

Reply:
Changed the content for customization.